src/MidnightExample/HSLedger.agda

```diff
@@ -30,10 +28,11 @@ instance
   Hashable-× : Hashable₂ _×_ Hash
   Hashable-× .hash (a , b) = hash (N.show (hash a) ++ "," ++ N.show (hash b))
 
-open import MidnightExample.Ledger Hash
+open import MidnightExample.Ledger Hash using (Point; Tx; inc; dec; Header; Block; LedgerState; LEDGER-step; addBlockHash)
+
+open import Foreign.Convertible using (Convertible; to; from)
 
-open import Foreign.Convertible
-open import Foreign.Haskell.Coerce
+-- open import Foreign.Haskell.Coerce -- (It seems we're not using this...?)
```

Comment:
In that case just go ahead and remove it. I don't think there's ever a reason not to remove an unused import.